### PR TITLE
#2182 Get rid of annoying popups in Windows 10

### PR DESCRIPTION
**Pull request by toddwong**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2182
| License       | MIT

Works with node v9.4.0 and above.

### CODE REVIEW

File: lib/God/ForkMode.js [hunk 1]
```diff
@@ -93,7 +93,6 @@ module.exports = function ForkMode(God) {
       try {
         var cspr = spawn(command, args, {
           env      : pm2_env,
-          detached : true,
```
Comment (soyuka) [inline lib/God/ForkMode.js]: do you really need to remove detached here?

Comment (toddwong) [reply lib/God/ForkMode.js]: @soyuka I can't make the windowsHide option works with `detached: true`. Do we actually needs detached to be true here? I maybe mistook something here, but child processes in cluster mode are attached to the daemon process I think.

Comment (toddwong) [reply lib/God/ForkMode.js]: @soyuka I tried to reproduce the problem (`detached:true` with 'windowsHide:true`) to find the real cause, but found something unexpected. I think the console windows may be not caused by this spawn call at all, but something else. I'll do some digging.

**Comment by toddwong** [comment]
@soyuka I've found the root cause of the popping up console window while start process in fork mode.  It's not the `spawn` call in lib/God/ForkMode.js, but `pmx.init` in lib/ProcessContainerFork.js. I've submit a PR to the pmx project, and undone the `detached: true` deletion.

**Comment by soyuka** [comment]
Great work @toddwong thanks!

**Comment by vmarchaud** [comment]
Tests failed are once again not linked to your PR, i will merge it when i finished with pmx :)

**Comment by toddwong** [comment]
Thanks,  @vmarchaud !

**Comment by soyuka** [comment]
ping @wallet77

**Comment by linonetwo** [comment]
Will this work in fork mode?

Cluster mode won't run ps1 file in windows. And fork mode still pop up cmd window.

**Pull request merged**